web/client/components/data/query/QueryToolbar.jsx

```diff
@@ -58,7 +59,8 @@ class QueryToolbar extends React.Component {
         sendFilters: {
             attributeFilter: true,
             spatialFilter: true,
-            crossLayerFilter: true
+            crossLayerFilter: true,
+            filters: true
```

Comment:
`sendFilters` is a boolean set of flags from query toolbar. It gets data from `attributePanelExpanded` and other flags like this.
I didn't find a utility for this flag at the moment, so I set it to true by default.

Reply:
Ok so it's wrong the proptype, because it is declared as array https://github.com/geosolutions-it/MapStore2/pull/8957/files/3215e686c6fbf53a234addcb2ee22813a332ad35#diff-e6ef7d2671afb4ccb42f545dc874643a65d9a77f708fea92ecf5855f1e5cdf70R31

Reply:
no. It is nested. 
sendFilters --> filters 

Send filter prototype is "object"

Reply:
ah sorry, just noticed the sendFilters prop